app/assets/stylesheets/welcome.scss

```diff
@@ -1,3 +1,52 @@
 // Place all the styles related to the welcome controller here.
 // They will automatically be included in application.css.
 // You can use Sass (SCSS) here: http://sass-lang.com/
+@font-face {
+  font-family: "Rubik";
+  src: url('/assets/Rubyk-Regular');
```

Comment:
I think you need to specify the extension of the file...
But I'd recommend you to get the font using Google fonts (https://fonts.google.com/specimen/Rubik), with that you can easily add all the fonts you need and they are also cached.